### PR TITLE
[T5A1][T17-B2] Abdul Rahman

### PR DESCRIPTION
**Pull request by arshrahman**
Ready for Review!~

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -47,7 +47,9 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
 `*` | user with many persons in the address book | sort persons by name | locate a person easily
-
+`*` | user| see suggestions as I type in the textbox | perform functions faster
```
Comment (JoelT-92) [inline doc/DeveloperGuide.md]: as **the user** types in the textbox

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -47,7 +47,9 @@ Priority | As a ... | I want to ... | So that I can...
...
+`*` | user| have auto-correct feature for the textbox | be more productive
+`*` | user| see all my contacts straightaway| find a person faster
```
Comment (JoelT-92) [inline doc/DeveloperGuide.md]: see all **the** contacts straightaway

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -74,12 +76,36 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename a specific tag with a new name
```
Comment (JoelT-92) [inline doc/DeveloperGuide.md]: Could be more detailed:

1. User requests to rename a tag
2. System asks for confirmation
3. User confirms the change
4. System renames/changes the tag for all entries that have the given tag.

Extensions:
1a. Tag not found
- 1a1. System informs the tag is not found
- Use case end.

1b. New name clashes with another existing tag
- 1b1. System informs user of the clash
- Use case ends.

1c. New name same as original tag
- 1c1. System informs no change was done.
- Use case ends.

2a. User rejects/cancels the change
- Use case ends.

Comment (arshrahman) [inline doc/DeveloperGuide.md]: Thank you very much!

**Pull request closed without merging**